src/init/features/functions/index.js

```diff
@@ -22,35 +24,69 @@ module.exports = function(setup, config) {
 
   setup.functions = {};
   var projectId = _.get(setup, "rcfile.projects.default");
+  var requiredApis = ["cloudfunctions.googleapis.com", "runtimeconfig.googleapis.com"];
   var enableApis;
   if (projectId) {
     enableApis = requireAccess({ project: projectId }, [scopes.CLOUD_PLATFORM]).then(function() {
-      enableApi(projectId, "cloudfunctions.googleapis.com");
-      enableApi(projectId, "runtimeconfig.googleapis.com");
+      return promiseAllSettled(requiredApis.map((api) => enableApi(projectId, api)));
```

Comment:
I think fixing these missing returned promises is a very important change that can be de-coupled from the conversation of the logic below. Once it's de-coupled, `promiseAllSettled` wouldn't be necessary either, as `Promise.all` can be used. Does that seem reasonable?

Reply:
Yeah it would be better if it's decoupled from here. I will look more into the code, and see how to properly handle such missing promises.